blocks/editable/format-toolbar.js

```diff
@@ -151,17 +152,24 @@ class FormatToolbar extends Component {
 						className="editable-format-toolbar__link-modal"
 						style={ linkStyle }
 						onSubmit={ this.submitLink }>
+						<label
```

Comment:
@mehigh there's no need to ask around, we know the stuff 😉  what I've said is that _a few years ago_ ARIA was not fully supported yet, today it is. We should also consider that a percentage of users might still use old screen reader versions since, especially the commercial ones like JAWS, they are pretty expensive. However, I'm OK with using aria-label _but_ if you want to learn more you should really read the 5 rules of ARIA usage:
https://www.w3.org/TR/using-aria/
First one:
> If you can use a native HTML element [HTML51] or attribute with the semantics and behaviour you require already built in, instead of re-purposing an element and adding an ARIA role, **state or property** to make it accessible, then do so.

That'e because ARIA shouldn't ever be used as something to "magically" fix things: always prefer native HTML instead.